docker-compose.yml

```diff
@@ -78,6 +79,7 @@ services:
       DJANGO_LOG_SERVER_PORT: 80
       no_proxy: clickhouse,grafana,vector,nuclio,opa,${no_proxy:-}
       NUMPROCS: 1
+      COVERAGE_PROCESS_START:
```

Comment:
Did you consider defining this env variable in docker-compose.dev.yml as it was done with COVERAGE_ENABLED?